plugins/proofpoint_tap/requirements.txt

```
@@ -1,5 +1,6 @@
 # List third-party dependencies here, separated by newlines. 
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
-html-table-parser-python3==0.1.5
-urlextract==0.14.0
\ No newline at end of file
+html-table-parser-python3==0.3.1
+urlextract==1.8.0
+parameterized==0.9.0
```

I just suggest using the parameterized in version `0.8.1`. Sometimes unittests may be failing due to the newest version which is `0.9.0`. If everything works fine in that case then leave it as it is.
```suggestion
parameterized==0.8.1
```